announcements/1.15.0.md

```diff
@@ -0,0 +1,72 @@
+* [Tracing support](https://anbox-cloud.io/docs/ref/anbox-https-api#heading--10tracing) in Anbox now includes support for tracing the WebRTC stack as well.
+* A new HTTP `/1.0/platform` API allows runtime configuration of the currently loaded platform. For WebRTC, you can now enable RTP trace collection and retrieve information about the current status of the streamer.
+* Anbox now uses the recently released [LXC 5.0](https://discuss.linuxcontainers.org/t/lxc-5-0-lts-has-been-released/14381).
+* The WebRTC streamer now allows streaming only audio, only video or neither of them, which can be useful in cases where only the exchange of OOB messages is needed. See the [JS SDK](https://github.com/anbox-cloud/anbox-streaming-sdk/blob/master/js/anbox-stream-sdk.js) <!-- wokeignore:rule=master --> for details on how to use this feature.
+* Anbox now uses [`jemalloc`](https://github.com/jemalloc/jemalloc) as memory allocator to improve its overall memory footprint.
+* A new OOB version 2 implementation allows bi-directional communication with the Anbox container through the use of data channels. See [the documentation](https://anbox-cloud.io/docs/howto/stream/oob-data) for more details. With the introduction of OOB version 2, the former implementation is deprecated and will be removed in a future Anbox Cloud release.
+
+### Android
+
+* Added Android security updates for August 2022 (see [Android Security Bulletin - August 2022](https://source.android.com/docs/security/bulletin/2022-08-01) for more information).
+* Updated the Android WebView to [103.0.5060.129](https://chromereleases.googleblog.com/2022/07/chrome-for-android-update_01510389319.html).
+* Updated the Mesa graphics driver stack to [22.0.5](https://docs.mesa3d.org/relnotes/22.0.5.html).
+* The `ashmem` kernel module is now optional for all supported Android versions. Anbox Cloud will drop support for `ashmem` with the 1.18 release.
+
+### Anbox Stream Gateway / Agent
+
+* Updated dqlite to its latest [1.11.1 release](https://github.com/canonical/dqlite/releases/tag/v1.11.1).
+
+### Appliance
+
+* The appliance now supports deployment on Ubuntu 22.04.
+* The minimum required storage size is now 20 GB.
+
+### Charms
+
+* On Ubuntu 22.04, the NVIDIA driver is now installed from the Ubuntu archive on arm64 systems instead of from the NVIDIA CUDA archive.
+
+### Anbox Cloud tests
+
+* The WebRTC benchmark tool (`anbox-cloud-tests.benchmark`) can now:
+  * Attach to existing sessions (`--session=<id>`).
+  * Mark sessions to be kept alive after the benchmark is done (`--keep-session`).
+  * Set the WebRTC log level to gain low-level insight (`--log-level=<level>`).
+
+## Known issues
+
+n/a
+
+## Bug fixes
+
+* AC-1036 A native crash happens when streaming from the camera on Safari (iOS device)
+* AC-1034 Fix race condition between `init.ranchu.rc` and SurfaceFlinger causing a crash
```

Comment:
Can you update the capitalisation of SurfaceFlinger in the wordlist?
This capitalisation here is correct. That might cause other failures, but I can fix them on the 1.15 branch.